guide/how-it-works/liquidity.md

```diff
@@ -306,6 +306,49 @@ Consider how you can help the user with channel management without them even kno
 
 By combining the business incentives of an LSP, clever engineering, and good design, you can build a bitcoin product that makes using the lightning network very easy for the user.
 
+## Channel reserve
+
+### What is a channel reserve?
+
+A channel reserve is an amount that is set aside by each channel participant which ensures neither have 'nothing at stake' if a cheating attempt occurs. This reserve can not be spent, and is held aside for the entirety of the channels lifetime.
+
+Channel reserves make cheating attempts less economical. When one channel party attempts to cheat the other and they are caught, a [penalty transaction](https://fiatjaf.com/73095980.html) can be used to steal all the cheating parties bitcoin as punishment. Channel reserves makes it so cheating attempts always have something at stake making this less likely to occur.
+
+### What is the reserve amount?
+
+The channel reserve amount is dynamic and unique to each channel participant. As defined in [BOLT 2](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md), the channel reserve amount dynamically trends towards 1% of the users local channel capacity. The channel reserve can not be lower than the current 354 sats minimum ([dust limit](https://github.com/lightning/bolts/blob/master/03-transactions.md#dust-limits)).
+
+So if a user has 100,000 sats of local capacity, their channel reserve will be 1000 sats (1% of 100,000). The user can only spend 99,000 sats of the local capacity. The channel counter party also has their own channel reserve which aims for a 1% reserve. This means the total channels capacity will have around 2% put aside and unspendable as a reserve.
+
+As users send and receive funds the channel reserve will dynamically adjust so its always close to 1%. If a users spends 10,000 sats of their 100,000 sats local capacity, their new channel reserve will dynamically adjust from 1000 sats (1% of 100,000 sats) to 900 sats (1% of 90,000). This adjust upwards if the user receives funds.
```

Comment:
```suggestion
As users send and receive funds, the channel reserve will dynamically adjust so that it's always close to 1%. If a user spends 10,000 sats of their 100,000 sats local capacity, their new channel reserve will dynamically adjust from 1,000 sats (1% of 100,000 sats) to 900 sats (1% of 90,000). This amount adjusts upwards if the user receives funds.
```